Translation/en/RedirectedResources/assets/abdata/communication/info_70/topiclistend_42/translation.txt

```diff
@@ -1,39 +1,39 @@
-//「あんな不思議な形なのって、やっぱり暑いから……だったり？」=
-//「<color%3Dyellow>ハイビスカス</color>って綺麗なだけじゃなくて、お茶にして飲んだり食用になるのがあるらしいよー」=
-//「私は観賞用のハイビスカスをずっと眺めていたいかなー」=
-//「あ、そっちもいいね。自然をこんなに感じられるのも、南の島の特権かもねー？」=
-//「そういうのもいいかも。本当に自然っていいなぁ」=
-//「あれ？　あんまり自然に興味なかったかなー？」=
-//「いまいち好みが合わないね？　ちょっと残念かも」=
-//「<color%3Dyellow>サンゴ</color>って長い時間をかけて少しずつ成長しているらしいね」=
-//「そんなに頑張って成長しているんだから、綺麗なのも当然なのかもー」=
-//「<color%3Dyellow>シオマネキ</color>って、どうして左右のハサミの大きさがあんなに違うんだろ？」=
-//「しかも、雄と雌でも大きさが違うみたいなんだよー。不思議だよねっ」=
-//「うんうん。海って、本当にいろいろな生き物がいてすごいよね」=
-//「もしかして、色々知ってるのかな？　私に教えてくれるー？」=
-//「あんまり海に興味がなかった？　ごめんね、つまらなかったかなー」=
-//「うーん、あんまり話があわないね？　残念かも」=
-//「欲しい<color%3Dyellow>お土産</color>がいっぱいありすぎて、お金が全然足りなくなっちゃいそうだよぉ」=
-//「星の砂とか、お菓子とか、キミも欲しいと思うよねー？」=
-//「<color%3Dyellow>シークヮーサージュース</color>に砂糖を少し入れて飲んでみて？　すっごく美味しいから！」=
-//「酸っぱさも強いから、口の中がきゅーってなっちゃうけどね、えへへ」=
-//「<color%3Dyellow>三線</color>の音色、独特な響きがあって私は好きだなー」=
-//「さすが名物って言われるだけの楽器だよね。カッコよく弾けるようになりたいな～」=
-//「名物っていうだけあって、どれもすっごい素敵だよねっ」=
-//「うぅぅ、あれもこれもって気になって来ちゃうよー」=
-//「他の話の方がいい？　そっかぁ、こっちもいいと思うんだけどなー」=
-//「興味なかったー？　じゃあ、今度はキミが興味を持ってるのを教えて欲しいな」=
+「のんびりと<color%3Dyellow>釣り</color>糸を垂らすのも、たまには良いと思うよー？」=“”
+「ゆっくりとした時間を過ごせて、釣ったお魚は食べられる。一石二鳥だねっ」=“It's such a nice and peaceful way to spend time, not to mention you can eat the fish you catch. Two birds with one stone, isn't it--”
+「海上とかで遊ぶのが好きなら、<color%3Dyellow>バナナボート</color>がおすすめだよ」=“”
+「滑るように海の上を走って、水しぶきを浴びながら風を感じるのが最高なんだー」=“You can experience the feeling of almost running on top of the water surface, and you get to feel the wind while water splashes on you from all over, it's the best~”
+「<color%3Dyellow>カヤック</color>に乗って、２人で競走するとかどうかなー？」=“”
+「たまには競走してみるのも楽しそうだよねっ」=“Having a race sounds fun, doesn't it--”
+「それも面白いよね。レジャーをこんなに楽しめるなんて、南の島は最高だよー」=“That sounds fun too. I can't believe we get to enjoy so many times, this island's the best~”
+「キミはそっちの方がいいのかな？　今度一緒にするのも良いかもね」=“You like that sorta stuff? Maybe we should do it together sometime.”
+「興味はなかったかなー？　残念だけど、自分がしたいことをするのが１番だよっ」=“Not interested~? It's a shame but, you should always do what you find the most fun anyway--”
+「もし少しでも気になったら、ぜひとも一度くらいためしてみてねー」=“If you've got even a bit of interest in it, do make sure to give it a shot atleast once, okay~?”
+「<color%3Dyellow>マングローブ</color>って熱帯や亜熱帯でしか見られないって本当なのかなー」=“”
+「あんな不思議な形なのって、やっぱり暑いから……だったり？」=“I mean I guess that's why they're shaped like that, it must be the heat... right?”
+「<color%3Dyellow>ハイビスカス</color>って綺麗なだけじゃなくて、お茶にして飲んだり食用になるのがあるらしいよー」=“”
+「私は観賞用のハイビスカスをずっと眺めていたいかなー」=“I'd much rather look at them all day instead though~”
+「あ、そっちもいいね。自然をこんなに感じられるのも、南の島の特権かもねー？」=“Ah, that sounds nice too. Being able to enjoy nature so much might just be one of the privileges of being here~”
+「そういうのもいいかも。本当に自然っていいなぁ」=“That sounds nice too. Nature really is just so nice~”
+「あれ？　あんまり自然に興味なかったかなー？」=“Huh? Does nature not interest you that much?”
+「いまいち好みが合わないね？　ちょっと残念かも」=“Our tastes don't match that much, huh? That's a bit of a shame.”
+「<color%3Dyellow>サンゴ</color>って長い時間をかけて少しずつ成長しているらしいね」=“”
```

Comment:
Looks like all lines with `<color>` tags are corrupted.  My have missed this is other reviews